.github/workflows/nestml-build.yml

```diff
@@ -227,7 +227,7 @@ jobs:
           done;
           exit $rc
 
-      # Run only the nest_integration_test for the NEST versions other than master and 2.20.2
+      # Run only the nest integration tests for NEST versions other than master and 2.20.2
```

Comment:
Why is this changed? We are indeed only running the `nest_integration_test` for versions other than master and 2.20.2.

Reply:
it's a stray, tiny improvement that is unrelated to the PR, mea culpa!